### PR TITLE
Address Zone Component Load Summary - DOAS Direct to Zone Output Zone Multiplier Issue

### LINKED ISSUE #9948: Zone Component Load Summary - DOAS Direct to Zone
Issue overview
--------------
If a zone multiplier (eg. 5) is used, the output file section Zone Component Load Summary; DOAS Direct to Zone is reported as multiplied by the multiplier (eg. 5x).  All other values are reported based on a multiple of 1.

### Details
Some additional details for this issue (if relevant):
 - Windows 11 Pro
 - EnergyPlus, Version 22.2.0-aa78da9668, YMD=2023.04.11 13:01

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by Nigusse**
…ions skip]

Pull request overview
---------------------
 - Fixes #9948
 - This fix removes the zone multiplier to `DOAS Direct to Zone` row in `Estimated Cooling Peak Load Components` table of Zone Component Load Summary report. It should have been reported without the zone multiplier just like the other components in the same table. 
 - This fix may cause diffs in the tabular output file.
 
**NOTE: ENHANCEMENTS MUST FOLLOW A SUBMISSION PROCESS INCLUDING A FEATURE PROPOSAL AND DESIGN DOCUMENT PRIOR TO SUBMITTING CODE**

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [x] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [x] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [x] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/OutputReportTabular.cc [hunk 1]
```diff
@@ -15595,9 +15595,12 @@ void ComputeTableBodyUsingMovingAvg(EnergyPlusData &state,
...
 
         // DOAS
-        resultCells(LoadCompCol::SensInst, LoadCompRow::DOAS) = state.dataSize->CalcZoneSizing(desDaySelected, zoneIndex).DOASHeatAddSeq(timeOfMax);
+        Real64 const mult = state.dataHeatBal->Zone(zoneIndex).Multiplier * state.dataHeatBal->Zone(zoneIndex).ListMultiplier;
+        resultCells(LoadCompCol::SensInst, LoadCompRow::DOAS) =
+            state.dataSize->CalcZoneSizing(desDaySelected, zoneIndex).DOASHeatAddSeq(timeOfMax) / mult;
         resCellsUsd(LoadCompCol::SensInst, LoadCompRow::DOAS) = true;
-        resultCells(LoadCompCol::Latent, LoadCompRow::DOAS) = state.dataSize->CalcZoneSizing(desDaySelected, zoneIndex).DOASLatAddSeq(timeOfMax);
+        resultCells(LoadCompCol::Latent, LoadCompRow::DOAS) =
+            state.dataSize->CalcZoneSizing(desDaySelected, zoneIndex).DOASLatAddSeq(timeOfMax) / mult;
```
Comment (Nigusse) [inline src/EnergyPlus/OutputReportTabular.cc:15603]: In the `Estimated Cooling Peak Load Components` table of `Zone Component Load Summary` report the `DOAS Direct to Zone` entry row is the only one scaled by zone multiplier. Other entries such as `People`, `Lights`, and `Equipment` peak load components are not multiplied by zone multiplier. This change is intended to fix the issue.  @mjwitte do you agree with this proposed change before I proceed further?

Comment (mjwitte) [reply src/EnergyPlus/OutputReportTabular.cc:15603]: Yes, `state.dataSize->CalcZoneSizing(desDaySelected, zoneIndex).DOASLatAddSeq(timeOfMax)` needs to be divided by `mult`.

Comment (Nigusse) [reply src/EnergyPlus/OutputReportTabular.cc:15603]: @mjwitte Thanks.

**Comment by Nigusse** [comment]
The `Estimated Cooling Peak Load Components` table before and after the fix is shown below. In this branch the `DOAS Direct to Zone` zone load component is not scaled by the zone multiplier. This branch's values are now divided by the zone multiplier value of `5` for consistency with the other peal load components of a zone.

Develop:
image

Branch:
image

File: tst/EnergyPlus/unit/OutputReportTabular.unit.cc [hunk 2]
```diff
@@ -77,6 +77,7 @@
 #include <EnergyPlus/ElectricPowerServiceManager.hh>
 #include <EnergyPlus/FileSystem.hh>
 #include <EnergyPlus/General.hh>
+#include <EnergyPlus/GeneralRoutines.hh>
```
Comment (rraustad) [inline tst/EnergyPlus/unit/OutputReportTabular.unit.cc]: Why is this header needed here?

**Review by rraustad — APPROVED** [review]
This looks correct to me. Leaving to @mjwitte for final sign off.

File: tst/EnergyPlus/unit/OutputReportTabular.unit.cc (hunk 2 above)
Comment (Nigusse) [reply tst/EnergyPlus/unit/OutputReportTabular.unit.cc]: Earlier I tried a different unit test, which required calling a function in the `GeneralRoutines.cc`. It is not needed for the current unit test and should have been removed. I will do that shortly.

**Comment by Nigusse** [comment]
The regression test suite files do not show diffs because the non of the example files with the input field `Account for Dedicated Outdoor Air System` field in the `Sizing:Zone` object set to `yes` has any `zone multiplier`. Unless you have both these combination diffs will not be observed. But the defect file shows diffs.

**Review by mjwitte — APPROVED** [review]
Looks good. Ran unit tests, tested defect file with several multipliers. Merging.

**Pull request merged**